benchmark_db_terraform/modules/benchmark_db/service/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM fedora:36
+
+RUN dnf -y install dnf-plugins-core && \
+    dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo && \
+    dnf install -y \
+        aria2 zstd clang clang-devel cmake git bzr jq pkgconfig mesa-libOpenCL mesa-libOpenCL-devel opencl-headers ocl-icd ocl-icd-devel llvm wget hwloc hwloc-devel golang rust cargo s3cmd \
+        docker-ce docker-ce-cli containerd.io docker-buildx-plugin \
+        ruby ruby-devel make gcc && \
+    dnf clean all
```

Comment:
Are you sure all of these are required to list? It'd be nice to group them and elaborate which package is a dependency of which feature.